tests/app/questionnaire/routing/test_operators.py

```diff
@@ -76,6 +80,15 @@ def test_operation_greater_than(operands, expected_result):
     assert operator.evaluate(operands) is expected_result
 
 
+@pytest.mark.parametrize(
+    "operands, expected_result",
+    greater_than_and_less_than_operations_equals_test,
+)
+def test_operation_greater_than_same_number(operands, expected_result):
```

Comment:
I don't see why we would have the same test repeated, if greater_than_and_less_than_operations_equals_test (or not_equals as suggested above) and greater_than_and_less_than_operations need to be kept separate for other test, why just not add them together for this one. Can be done before the parametrize or even in it like 

```@pytest.mark.parametrize(
    "operands, expected_result",
    greater_than_and_less_than_operations + not_equals,
)
def test_operation_greater_than(operands, expected_result):
    operator = Operator(Operator.GREATER_THAN)
    assert operator.evaluate(operands) is expected_result
```

Reply:
Although this works for the greater than, how less than is implement it wouldn't. For me the tests still look odd and have repetition, but it works, so will resolve

Reply:
I would do this but combining the tests like this would only work for the greater than ones. For the less than ones, one is expecting the result and one is not expecting the result so it wouldn't work combining them. I could still combine the greater than one but I just think doing one and not the other would be more confusing.